### ISSUE TITLE
Read permissions not set when read_only_installdir or group_writable_installdir is used

### ISSUE THREAD

**Issue by akesandgren** [easybuilders/easybuild-framework#2769]
When either of read_only_installdir or group_writable_installdir is True the file permissions in installdir are not set to read for anyone.

This is counter-intuitive, one would assume that the above two have no effect on read permissions, that's the job of umask.

I think that in framework/easyblock.py the block starting with (line ~2535)
```
# add read permissions for everybody on all files, taking into account group (if any)
```
should be move outside of the
```
else: # remove write permissions for group and other
```
section.

**Comment by akesandgren** [comment]
Suggested fix in https://github.com/easybuilders/easybuild-framework/pull/2770

**Comment by boegel** [comment]
fixed in #2770
